### ISSUE TITLE
Single view template files require being ignored, results in lack of live reload

### ISSUE THREAD

**Issue by declandewet** [carrot/roots-records#16]
If one does not prepend an underscore to a single view `roots-records` template, the template is compiled as a regular view and this sometimes causes error because the plain roots context does not provide any of the `roots-records` variable locals. However, prepending an underscore so that roots ignores the file will also prevent live reload, forcing the developer to restart the roots watch process after making any changes to layout.

I have a hunch this is a simple fix that involves setting the `extract` option to true and matching the file name against the name provided to the options parameter. This fix could also include any local data changes to JSON files.

**Comment by jenius** [comment]
Yeah this seems like a valid issue. I personally can't close it out, but would love for someone to take a look into it!
